src/navsat_transform.cpp

```diff
@@ -310,6 +320,14 @@ namespace RobotLocalization
 
       utmWorldTransInverse_ = utmWorldTransform_.inverse();
 
+      // publish the origin of the world frame in NavSatFix format
+      sensor_msgs::NavSatFix nsf;
+      NavsatConversions::UTMtoLL(utmWorldTransInverse_.getOrigin().getY(), 
```

Comment:
So wild how you pass references to fill with the result of a function as you can't comfortably return arrays, dictionaries, maps, etc.  This catches me every time I read C++.